### DISCUSSION TITLE
Invalidate path: conflicting docs

### DISCUSSION THREAD

**Discussion by elie222** [vercel/next.js#52286] [App Router]
I'm seeing conflicting information about revalidating paths.

In this blog post I see that  `"revalidatePath(`/posts/${slug}`);"` should work.
https://vercel.com/blog/vercel-cache-api-nextjs-cache#revalidating-components-with-cache-tags

But from the docs I see it shouldn't work:
https://nextjs.org/docs/app/api-reference/functions/revalidatePath#parameters

> path: A string representing the filesystem path associated with the data you want to revalidate. This is not the literal route segment (e.g. /product/123) but instead the path on the filesystem (e.g. /product/[id]).

Which is correct? And it would be great if the former one did work. If I've updated a specific product, why would I want to invalidate all products?

**Comment by elie222** [comment]
Does the Vercel team know?
I struggled to get either working so can't say.

**Comment by nrsimha** [comment]
I would love too to be able to revalidate literal route segment e.g. /product/123456
Is it currently possible?

**Comment by leerob** [comment]
Hey there, apologies that the blog post is out of date. I'll fix that now. We're also exploring adding support for the literal routes 👍

**Reply by elie222** [reply]
Curious why that isn't the default. If you have a database of 10,000 projects, and one project is updated, why would you want to revalidate the cache for all 10,000? You'd want to immediately clear the cache for the single project so that it's not stale on next page load, and leave rest untouched.
